locales/base/translation.json

```diff
@@ -2244,6 +2244,15 @@
   "points": {
     "title": "Valora Points",
     "activity": "Activity",
+    "loading": {
+      "title": "Gathering your points",
+      "description": "We're in the process of tallying up your points, hang tight"
```

Comment:
nit: should this copy not have a period, unlike the error description? :eyes: i see that this matches the figma designs but not sure if it's intentional.

Reply:
i'll double check! i think the copy is not final yet though, seems reasonable to assume that we'll do a pass through before releasing this